modulemd/tests/test-modulemd-moduleindex.c

```diff
@@ -525,7 +499,7 @@ module_index_test_stream_upgrade (void)
   g_assert_no_error (error);
   g_assert_cmpint (modulemd_module_stream_get_mdversion (stream),
                    ==,
```

Comment:
The comment a few lines above should just say `/* Verify that it was added as StreamV1 */`